_i18n/en.yml

```diff
@@ -0,0 +1,163 @@
+  compilers:
+    title: Fortran Compilers
+  community:
+    title: Fortran-lang Community
+    description: Collaboration for the advancement of Fortran
+  packages:
+    title: Fortran Packages
+    description: A rich ecosystem of high-performance code
+    search:
+      title: Search
+    category:
+      data-types:
+        title: Data types and containers
+        description: Libraries for advanced data types and container classes
+      examples:
+        title: Examples and templates
+        description: Demonstration codes and templates for Fortran
+      fpm:
+        title: FPM Registry Packages
+        description: Packages listed in the fpm-registry
+      graphics:
+        title: Graphics, plotting and user interfaces
+        description: Libraries for plotting data, handling images and generating user interfaces
+      interfaces:
+        title: Interface libraries
+        description: Libraries that interface with other systems, languages, or devices
+      numerical:
+        title: Numerical projects
+        description: Fortran libraries for linear algebra, optimization, root-finding etc.
+      programming:
+        title: Programming utilities
+        description: Error handling, logging, documentation and testing
+      scientific:
+        title: Scientific Codes
+        description: Applications and libraries for applied mathematical and scientific problems
+      strings:
+        title: Characters and strings
+        description: Libraries for manipulating characters and strings
+      io:
+        title: Input, output and parsing
+        description: Libraries for reading, writing and parsing files and inputs
+      libraries:
+        title: Libraries
+        description: Fortran libraries for general programming tasks
+
+home:
+  tagline: High-performance parallel programming language
+  get_started: Get started
+  more: More
+  join_us: Join us!
+  make_fortran_better: Make Fortran better
+
+nav:
+- title: Learn
+  url: /learn/
+- title: Compilers
+  url: /compilers/
+- title: Community
+  url: /community/
+- title: Packages
+  url: /packages/
+- title: News
+  url: /news/
+
+nav_key:
+  home: Home
+  learn: Learn
+  compilers: Compilers
+  community: Community
+  packages: Packages
+  news: News
+
+learn:
+  getting_started: Getting started
+  quickstart_tutorial: Quickstart tutorial
+  fortran_lang_discourse: Fortran-lang Discourse
+  minibook_tutorials: Mini-book Tutorials
+  other_resources: Other Resources
+  on_the_web: On the web
+  in_print: In print
+
+book:
+  authors: Authors
+  introduction: Introduction
+  back_to_learn: Back to Learn Fortran index
+  important: Important
+  note: Note
+  tip: Tip
+
+book_nav:
+  back: Back
+  next: Next
+
+news:
+  archive: Archive
+  authors: Authors
+  back_to_news: Back to news
+  more: More
+  read_more: Read more
+  see_archive: >-
+    See the
+    <a href="{{ site.baseurl }}/news/archive">news archive</a> for older posts
```

Comment:
It think this substitution (News archive link at bottom of news page) isn't working in the PR preview

Reply:
FIxed